pkg/kubeapiserver/admission/initializer.go

```diff
@@ -67,6 +75,14 @@ func (i pluginInitializer) Initialize(plugin admission.Interface) {
 		wants.SetInternalClientSet(i.internalClient)
 	}
 
+	if wants, ok := plugin.(WantsToRun); ok {
```

Comment:
@deads2k -- so we need some change here to support cluster capacity tool if i recall being able to simulate admission, i am fine with https://github.com/kubernetes/kubernetes/pull/37148#discussion_r95996153 if that is how you want to proceed (not sure if you think that still gels with what your last comment noted)